app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -593,6 +605,9 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
     pointerTransform
         .PostMultiplyInPlace(vrb::Matrix::Rotation(vrb::Vector(0.0, 1.0, 0.0),correctionAngle)
         .PostMultiply(vrb::Matrix::Rotation(vrb::Vector(0.0, 0.0, 1.0), correctionAngle)));
+#elif defined(LYNX)
```

Comment:
I trust you on this logic :)

Reply:
It's just basically adding some rotation. The values are indeed taken from experiments so yeah I guess you'd have to trust me :smile: